core/src/Kokkos_HIP_Space.hpp

```diff
@@ -738,6 +739,20 @@ struct DeviceTypeTraits<Kokkos::Experimental::HIP> {
 };
 }  // namespace Experimental
 }  // namespace Tools
+
+namespace Experimental {
+
+class HIPSpaceInitializer : public Kokkos::Impl::ExecSpaceInitializerBase {
```

Comment:
Shouldn't this be in namespace `Impl` for consistency?

Reply:
Addressed in 1cb82e0